README.md

```diff
@@ -108,9 +108,11 @@ Or use the 1-click deployment button
 
  - Once deployed, you'll find the generator running on http://localhost:5000, it should look like this  
 
-![Generator Screencast](docs/screenshots/screencast.gif)
+![Generator Screencast](docs/screenshots/openevent1.gif)
 
- - Add your **email id**, **name of app** (name of event),
+srijanjha09@gmail.com
```

Comment:
It is not required for you to actually enter your email here.
If you read the Readme carefully, these are the steps which guide the user on how to use this generator.